.travis.yml

```diff
@@ -3,18 +3,21 @@ os:
   - linux
   - osx
 julia:
-  - 1.0
   - 1.4
   - nightly
 codecov: true
 jobs:
   allow_failures:
     - julia: nightly
   include:
-    - stage: "Documentation"
+    - stage: "Test"
       julia: 1.4
       os: linux
       script:
-        - julia --project=docs/ -e 'using Pkg; Pkg.instantiate(); Pkg.develop(PackageSpec(path=pwd()))'
-        - julia --project=docs/ docs/make.jl
+        - julia --project=. -e '
+            using Pkg; 
+            Pkg.instantiate(); 
+            Pkg.add("Coverage");
+            using Coverage;
+            Codecov.submit(process_folder());'
```

Comment:
I'm confused what this step does other than regular testing, which the julia block above should do. If you need coverage, that should already happen automatically with the original code. Also, we don't want to remove the documentation build step.